src/styles/page.scss

```diff
@@ -0,0 +1,91 @@
+.home-page {
+  box-sizing: border-box;
+  font-family: "Roboto", sans-serif;
+  font-weight: 500;
+  font-size: 12px;
+  --main-color: #060b35;
+  --second-color: #616070;
+}
+
+body,
+.nav__list {
+  margin: 0;
+  padding: 0;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  background: #fff;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  padding: 0 50px;
+}
+
+.nav {
+  display: flex;
+  align-items: center;
+  justify-content: right;
+  position: relative;
+
+  &__list {
+    display: flex;
+    list-style: none;
+  }
+
+  &__item:not(:last-child) {
+    margin-right: 20px;
+  }
+
+  &__link {
+    display: flex;
+    align-items: center;
+    text-decoration: none;
+    color: var(--main-color);
+    font-size: 12px;
+    text-transform: uppercase;
+    padding: 23px 0;
+    position: relative;
+
+    &:hover {
+      color: #00acdc;
+      transition: color 0.5s;
+    }
+  }
+}
+
+.img__logo {
+  display: flex;
+  width: 40px;
+  height: 40px;
+  padding: 0;
+  justify-self: center;
+  margin: 10px 0;
+}
+
+.is-active {
+  color: #00acdc;
+}
+
+.is-active::after {
+  position: absolute;
+  content: "";
+  height: 4px;
+  background-color: #00acdc;
+  width: 100%;
+  border-radius: 8px;
+  bottom: 0;
+  left: 0;
+}
```

Comment:
is-active {
  color: #00acdc;

  &::after {
    position: absolute;
    content: "";
    height: 4px;
    background-color: #00acdc;
    width: 100%;
    border-radius: 8px;
    bottom: 0;
    left: 0;
  }
}